File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/get-person-vetting-vetting-type_item.ts


import { IGetPersonVettingVettingType_Item } from './iget-person-vetting-vetting-type_item';

export class GetPersonVettingVettingType_Item implements IGetPersonVettingVettingType_Item {
  
	public FirstMiddleNames: string = "";
	public LastNames: string = "";
	public PersonsVettingID: number = 0;
	public VettingTypeID?: number;
	public VettingTypeCode: string = "";
	public CourtesyVettingSkipped?: boolean;
	public CourtestVettingSkippedComments: string = "";
  
}

File: GTTS/INL.VettingService.Models/GetPersonVettingVettingType_Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.VettingService.Models
{
    public class GetPersonVettingVettingType_Item : IGetPersonVettingVettingType_Item
    {
        public string FirstMiddleNames { get; set; }
        public string LastNames { get; set; }
        public long PersonsVettingID { get; set; }
        public int? VettingTypeID { get; set; }
        public string VettingTypeCode { get; set; }
        public bool? CourtesyVettingSkipped { get; set; }
        public string CourtestVettingSkippedComments { get; set; }
    }
}
